addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -342,6 +342,37 @@ def _import_retrieve_and_fill_partner(self, invoice, name, phone, mail, vat, cou
             if vat and self.env['res.partner']._run_vat_test(vat, country, invoice.partner_id.is_company):
                 invoice.partner_id.vat = vat
 
+    def _import_retrieve_and_fill_partner_bank_details(self, invoice, bank_details):
+        if invoice.move_type == 'in_invoice':
+            partner = invoice.partner_id
+        elif invoice.move_type == 'out_invoice':
+            partner = self.env.company.partner_id
+        else:
+            return
```

Comment:
So no bank imported for the refunds :'(
I'd rather use the journal type as in `_import_invoice` :)